cnd/src/storage.rs

```diff
@@ -1,18 +1,13 @@
 use crate::{
     asset,
     db::{
-        self,
-        tables::{Halight, Hbit, Herc20, SecretHash, Swap},
-        wrapper_types::custom_sql_types::Text,
-        ForSwap, NoHalightRedeemIdentity, NoHalightRefundIdentity, NoHerc20RedeemIdentity,
-        NoHerc20RefundIdentity, NoSecretHash, Save, Sqlite,
+        self, ForSwap, Halight, Hbit, Herc20, NoHalightRedeemIdentity, NoHalightRefundIdentity,
```

Comment:
:thinking: This patch doesn't introduce the problem, but I find the names `Halight`, `Hbit` and `Herc20` too general. Even if you're in the context of the `storage` module. I would find it clearer if we used them with a more qualified name, e.g `table::Halight` or even `db::table::Halight` (sorry I like long names [sometimes]).

Reply:
`db::<protocol-name>` would be better, but still a bit lacking imo.

Reply:
I have to disagree with you here mate. Apart from the fact that I always favour short names; the storage module is different in that it fully 'owns' the database, so bringing types from the db into scope is the natural thing to do IMO. The fact that the new tables are in the tables module is not important, when I'm writing/reading code in the storage module it is just like we are in the db layer so all the types are implicitly db types. Does that make sense? We don't have to agree :) (on this point, that's why I originally brought `SecretHash` into scope from the db and used `comit::SecretHash` when needed, you did it the other way around and I left it as such when I rebased :)

Reply:
Yeah, I'm okay with this even if I don't love it :wink:

Reply:
> the storage module is different in that it fully 'owns' the database

Would be nice to express that in our module hierachy.

Reply:
Oh, I like that.

`db` and `seed` being private scope modules under `storage`. Will wait for the remove of rfc003 before doing this, adding to my TODO list.